src/utils/utility-types.d.ts

```diff
@@ -0,0 +1,7 @@
+type AllKeys<T> = T extends unknown ? keyof T : never;
```

Comment:
I'd use the `src/utils/types.tsx` file.

Also, you don't need any change in `js-toolbox` if you follow [the same approach we did with the `RendersElement`](https://github.com/Telefonica/mistica-web/pull/413) utility type: you can define an override for flow inside the `flow-overrides/` folder.